schema/network.yaml

```diff
@@ -68,6 +68,12 @@ definitions:
         type: string
         readOnly: true
         description: Indicate the interface name.
+      status:
+        type: boolean
+        description: Connection status of the interface, true for connected.
+      wan:
+        type: boolean
+        description: Indicate the interface worked as WAN or not (LAN).
```

Comment:
A WiFi interface may be in client mode or AP mode. The interface cannot be WAN if in AP mode.